content/en/platform/corda/5.0-beta/managing-application-networks/creating/overview.md

```diff
@@ -0,0 +1,13 @@
+---
+date: '2023-04-18'
+title: "Creating Application Networks"
+menu:
+  corda-5-beta:
+    identifier: corda-5-beta-app-networks-create
+    parent: corda-5-beta-app-networks
+    weight: 3000
+section_menu: corda-5-beta
+---
+This section describes how to create a dynamic network. The network requires a running {{< tooltip >}}MGM{{< definition term="MGM" >}}{{< /tooltip >}} that all members must register with before they can transact among the group. This section also describes how to onboard a new member as a {{< tooltip >}}notary{{< definition term="Notary" >}}{{< /tooltip >}} service representative. 
```

Comment:
I had originally designed it like that but I wanted to allow for cases where we might want to use a different case or the plural version of what is actually defined in the JSON file. 
I did try to ignore the case when comparing but ran into some issues with the way Hugo reads in the JSON. I kind of thought that since this is only a temporary solution until we move away from Hugo it wasn't worth spending any longer on.

Reply:
Can you define the `tooltip` to be more flexible ... IOW ... if not given `definition term` it will by default take the term MGM (as in this example), otherwise, to serve the use-cases you mentioned ... it will rely on the `definition term` value.
I believe in most cases the tech-writers will not have to define `definition term`

Reply:
Definitely worth investigating, thanks! Have opened a new ticket (https://r3-cev.atlassian.net/browse/DOC-5257) to look into that. Thanks